spark/dl/src/main/scala/com/intel/analytics/bigdl/python/api/PythonBigDL.scala

```diff
@@ -1991,14 +1991,40 @@ class PythonBigDL[T: ClassTag](implicit ev: TensorNumeric[T]) extends Serializab
     layer.setInitMethod(weightInitMethod, biasInitMethod)
   }
 
-  def getHiddenStates(rec: Recurrent[T]): JList[JTensor] = {
+  def getHiddenStates(rec: Recurrent[T]): JList[JList[JTensor]] = {
```

Comment:
This is too complex; instead of doing this, maybe we can have automatic mapping of Table to JList?

Reply:
@jason-dai  issue created for this: https://github.com/intel-analytics/BigDL/issues/1937